README.md

```diff
@@ -2,143 +2,144 @@
 
 > A curated list of awesome things related to [Deno](https://github.com/denoland/deno).
 
-- [Docs](#docs)
-    - [Official Docs](#official-docs)
-    - [External Docs](#external-docs)
-- [Modules](#modules)
-- [Tools](#tools)
-- [Articles](#articles)
-- [Presentations](#presentations)
-- [Newsletters](#newsletters)
-- [Resources in Other Languages](#resources-in-other-languages)
-  - [Chinese](#chinese)
-  - [Hebrew](#hebrew)
-  - [Japanese](#japanese)
-  - [Korean](#korean)
+-   [Docs](#docs)
```

Comment:
Could you keep the original formatting of entries?

Reply:
ok,done